x-pack/plugins/security_solution/public/common/components/endpoint/page_view.tsx

```diff
@@ -25,15 +25,17 @@ import { EuiTabProps } from '@elastic/eui/src/components/tabs/tab';
 
 const StyledEuiPage = styled(EuiPage)`
   &.endpoint--isListView {
-    padding: 0 ${(props) => props.theme.eui.euiSizeL};
+    padding: 0;
+    flex: 1 1 100%;
 
     .endpoint-header {
       padding: ${(props) => props.theme.eui.euiSizeL};
       margin-bottom: 0;
     }
     .endpoint-page-content {
-      border-left: none;
-      border-right: none;
+      border: none;
+      box-shadow: none;
+      border-radius: 0;
```

Comment:
I discussed with Kevin that it feels like we use EuiPage component but then we actually overwrite a lot of it's styling so it's not clear if that component is helping much. I think it would be nice to have this type of layout supported by EUI.